src/Sarif/ExtensionMethods.cs

```diff
@@ -16,6 +16,11 @@ namespace Microsoft.CodeAnalysis.Sarif
 {
     public static class ExtensionMethods
     {
+        public static string Format(this IRule rule, string messageId, IEnumerable<string> arguments)
```

Comment:
>Format [](start = 29, length = 6)

This extension method created as a result of pushing IRule into the OM. Method previously existed on Rule, which is defined as a partial class. Can't do this (partial class) for an interface, obviously